From: Suren Baghdasaryan <surenb@google.com>
Subject: fixup! fix missing vmalloc.h includes
Date: Tue, 2 Apr 2024 11:09:32 -0700

vmalloc.h header refactoring missed vmalloc.h inclusion required for
__vmalloc_start_set symbol.  Add missing inclusion.

Link: https://lkml.kernel.org/r/20240402180933.1663992-1-surenb@google.com
Fixes: 07aecea15cc8 ("fix missing vmalloc.h includes")
Signed-off-by: Suren Baghdasaryan <surenb@google.com>
Reported-by: kernel test robot <lkp@intel.com>
Closes: https://lore.kernel.org/oe-kbuild-all/202403290536.7f9zGl5Q-lkp@intel.com/
Cc: Kent Overstreet <kent.overstreet@linux.dev>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 arch/x86/mm/numa_32.c |    1 +
 1 file changed, 1 insertion(+)

--- a/arch/x86/mm/numa_32.c~fix-missing-vmalloch-includes-fix-2
+++ a/arch/x86/mm/numa_32.c
@@ -24,6 +24,7 @@
 
 #include <linux/memblock.h>
 #include <linux/init.h>
+#include <linux/vmalloc.h>
 
 #include "numa_internal.h"
 
_
